packages/@ember/-internals/glimmer/lib/resolver.ts

```diff
@@ -361,6 +370,11 @@ export default class RuntimeResolverImpl implements RuntimeResolver<Owner> {
         manager === SIMPLE_CLASSIC_HELPER_MANAGER
     );
 
+    assert(
+      'internal helpers are not supported via `getHelperManager` yet',
```

Comment:
should include more information here, for example it should include the debug render tree stack and the name of the helper they asked for

Reply:
The debug render tree stack will be included in general, because we throw an error. There's no reason to manually add the stack, it'll always be logged regardless. Will add the name.